beaker/session.py

```diff
@@ -548,8 +540,15 @@ def __init__(self, request, key='beaker.session.id', timeout=None,
                 cookie_data = self.cookie[self.key].value
                 self.update(self._decrypt_data(cookie_data))
                 self._path = self.get('_path', '/')
-            except:
-                pass
```

Comment:
this line was causing a huge amount of trouble for me - a large number of unexpected exceptions were getting silenced. notably, when self.invalidate_corrupt was checked inside beaker/session but didn't exist, it would throw an AttributeError that would get silenced here